MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/annotations/AnimatedMarkerActivity.java

```diff
@@ -44,31 +51,80 @@ protected void onCreate(Bundle savedInstanceState) {
   }
 
   @Override
-  public void onMapReady(MapboxMap mapboxMap) {
-    AnimatedMarkerActivity.this.mapboxMap = mapboxMap;
-
-    marker = mapboxMap.addMarker(new MarkerOptions()
-      .position(new LatLng(64.900932, -18.167040)));
-
-    Toast.makeText(
-      AnimatedMarkerActivity.this,
-      getString(R.string.tap_on_map_instruction),
-      Toast.LENGTH_LONG
-    ).show();
-
-    mapboxMap.addOnMapClickListener(this);
+  public void onMapReady(@NonNull final MapboxMap mapboxMap) {
+    this.mapboxMap = mapboxMap;
+
+    geoJsonSource = new GeoJsonSource("source-id",
+      Feature.fromGeometry(Point.fromLngLat(currentPosition.getLongitude(), currentPosition.getLatitude())));
```

Comment:
This can be simplified to using the `GeoJsonSource(String id, Geometry geometry)` constructor.

Reply:
Fixed to 

```
geoJsonSource = new GeoJsonSource("source-id",
      Point.fromLngLat(currentPosition.getLongitude(),
          currentPosition.getLatitude()));
```